src/pages/Dashboard.vue

```diff
@@ -1,29 +1,28 @@
 <template>
   <q-page-container>
-    <p>dsdsd</p>
+    <p>sasa</p>
     <q-btn @click="logout">Logout</q-btn>
   </q-page-container>
 </template>
 
 <script>
 import firebase from "firebase";
+import { mapGetters, mapActions } from "vuex";
 
 export default {
   name: "LayoutDefault",
   computed: {
-    ...mapGetters(["getUser", "isUserAuth"])
+    ...mapGetters("user", ["getUser", "isUserAuth"])
```

Comment:
@priyankashrma you can get rid off passing "user" every time like this, by removing the **namespaced:true** over here https://github.com/priyankashrma/quasar-login-signup/blob/master/src/store/user/index.js#L7